apps/server/src/entities/learnroom/dashboard.entity.ts

```diff
@@ -0,0 +1,72 @@
+import { BaseEntityWithTimestamps } from '@shared/domain';
+
+export type GridElementReferenceMetadata = {
+	id: string;
+	title: string;
+	shortTitle: string;
+	displayColor: string;
+};
+
+export interface IGridElementReference {
+	getMetadata: () => GridElementReferenceMetadata;
+}
+
+export class DefaultGridReference implements IGridElementReference {
+	// This is only a temporary fake class, for use until other references, like courses, are fully supported.
+	title: string;
+
+	constructor(title: string) {
+		this.title = title;
+	}
+
+	getMetadata(): GridElementReferenceMetadata {
+		return {
+			id: 'someId',
```

Comment:
for what is id and shortTitle used for?

Reply:
why is that an entity? is it persisted? what's its identity?